mesop/components/link/link.ts

```diff
@@ -0,0 +1,38 @@
+import {ChangeDetectionStrategy, Component, Input} from '@angular/core';
+import {
+  Key,
+  Style,
+  Type,
+  UserEvent,
+} from 'mesop/mesop/protos/ui_jspb_proto_pb/mesop/protos/ui_pb';
+import {LinkType} from 'mesop/mesop/components/link/link_jspb_proto_pb/mesop/components/link/link_pb';
+import {Channel} from '../../web/src/services/channel';
+import {formatStyle} from '../../web/src/utils/styles';
+
+@Component({
+  selector: 'mesop-link',
+  templateUrl: 'link.ng.html',
+  styleUrl: 'link.css',
+  standalone: true,
+})
+export class LinkComponent {
+  @Input({required: true}) type!: Type;
+  @Input() key!: Key;
+  @Input() style!: Style;
+  private _config!: LinkType;
+  isChecked = false;
```

Comment:
Is this needed?

Reply:
nope - it's from the boilerplate generator; turns out I left it in a bunch of other components (will clean up separately).